### ISSUE TITLE
Update phone number patterns for Morocco

### ISSUE THREAD

**Issue by KeepItSimpleStupid** [googlei18n/libphonenumber#1202]
Hello !

**Country/region affected**: Morocco (MA)
**The type of the number(s)**: fixed-line and mobile
**Supporting evidences**:
According to this [document](https://www.anrt.ma/sites/default/files/Blocs_de_numeros_par_operateur.pdf) updated late December 2015, which is already referenced as a source in a [comment of PhoneNumberMetadata.xml](https://github.com/googlei18n/libphonenumber/blob/master/resources/PhoneNumberMetadata.xml#L15525), the mobile phone number pattern for Morocco (and Western Sahara ?) should look like :
```
6(?:
  (?:
    [0-7]\d|
    8[0-247-9]
  )\d|
  9(?:
    [013-9]\d|
    2[12]
  )
)\d{5}
```
or if we consider the range 0692ABCDEF totally valid (like in the current pattern) and not only 06921ABCDE and 06922ABCDE (like the document states) :
```
6(?:
  [0-79]\d|
  8[0-247-9]
)\d{6}
```
***Added* :**
0609ABCDEF
068[2489]ABCDE
***Removed* :** 
0692[03-9]ABCDE

I also computed the pattern for fixed phone numbers according to the document :
```
5(?:
  2(?:
    (?:
      [015-7]\d|
      2[2-9]|
      3[2-57]|
      4[2-46-8]|
      8[235-9]
    )\d|
    9(?:
      0\d|
      [89]0
    )
  )|
  3(?:
    (?:
      [0-4]\d|
      [57][2-9]|
      6[235-8]|
      9[3-9]
    )\d|
    8(?:
      0\d|
      [89]0
    )
  )|
  (?:
    4[067]|
    5[03]
  )\d{2}
)\d{4}
```
***Added* :**
0528[89]ABCDE
054[067]ABCDEF
055[03]ABCDEF
***Removed* :** 
05220ABCDE
05245ABCDE

Thanks !

**Comment by penmetsaa** [comment]
@KeepItSimpleStupid 
Thank you for taking the time to improve libphonenumber!
We've accepted this issue and will fix this in an upcoming release.

We will be not removing 05220ABCDE and 05245ABCDE fixed-line ranges as we are able to connect to the numbers that are present online. http://www.telecontact.ma/annonceur/sekkat-jihane-/3297707/casablanca.php (Range 5200)
https://www.google.co.in/#q=%22%2B212+5245%22 (Range 5245)

It might be that the official doc is not up to date.
https://www.anrt.ma/sites/default/files/Blocs_de_numeros_par_operateur.pdf

With regards to remaining ranges, we will make changes as mentioned in the official doc.

Please note that you have submitted this information with the knowledge that we
may use, modify, reproduce, publicly display, and distribute the information in
accordance with our open source policies. Please let us know if you did not
intend to submit the information for such use.

**Comment by KeepItSimpleStupid** [comment]
Thank you @penmetsaa !

I just found an updated version (july 2016) of the doc on Morocco's telecom regulator website : https://www.anrt.ma/sites/default/files/blocs_de_numeros_par_operateur_juillet_2016.pdf

It states that the following mobile phone number ranges will be opened beginning of August :
0700ABCDEF
0707ABCDEF
0761ABCDEF
0762ABCDEF
0770ABCDEF
0777ABCDEF

Thanks !

**Comment by penmetsaa** [comment]
Thanks providing the new official document. We came across this doc in other GitHub ( #1220) issue as well.  We are planning to support the new doc in same update.

**Comment by KeepItSimpleStupid** [comment]
Good to know @penmetsaa. Let me know if you need something from me ! Thanks :-)

**Comment by keghani** [comment]
Fixed in 7.6.0.

**Comment by KeepItSimpleStupid** [comment]
Thanks !
